File: Day 20 : Sorting/Solution.py
if __name__ == '__main__':
    n = int(input())

    a = list(map(int, input().split()))

    numberOfSwaps = 0
    for i in range(n):
        for j in range(n-1):
            if a[j] > a[j+1]:
                a[j], a[j+1] = a[j+1], a[j]
                numberOfSwaps += 1
        
        if numberOfSwaps == 0:
            break
    
    print(f"Array is sorted in {numberOfSwaps} swaps.")
    print("First Element:", a[0])
    print("Last Element:", a[-1])

File: Day 20 : Sorting/Solution.js
function main() {
        const n = parseInt(readLine().trim(), 10);

        const a = readLine().split(' ');

        var numberOfSwaps = 0;
        for (let i = 0; i < n; i++) {
                for (let j = 0; j < n - 1; j++) {
                        if (a[j] > a[j + 1]) {
                                let temp = a[j];
                                a[j] = a[j + 1];
                                a[j + 1] = temp;
                                numberOfSwaps++;
                        }
                }

                if (numberOfSwaps == 0) {
                        break;
                }
        }

        console.log(`Array is sorted in ${numberOfSwaps} swaps.`);
        console.log(`First Element: ${a[0]}`);
        console.log(`Last Element: ${a[a.length - 1]}`);
}

main();
